js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -0,0 +1,19 @@
+function alternatingCharacters(arr) {
+    let deleteCountArray = [];
+
+    for (let k = 0; k < arr.length; k++) {
+        let c = '';
+        let deleteCount = 0;
+        for (let i = 0; i < arr[k].length; i++) {
+            if (c !== arr[k][i])
+                c = arr[k][i];
+            else
+                deleteCount++;    
+            }
```

Comment:
Logic for counting the number of deletions could be separated in different function.